Update dependencies and refactor ZoomApp image handling

Demontego opened the pull request:
- Added Pillow to requirements for image processing.
- Updated ZoomApp to use new image paths and improved element interaction.
- Modified MODULE.bazel to reflect the latest Zoom client version.
- Enhanced .gitignore to include temporary files.

Demontego committed:
Update dependencies and refactor ZoomApp image handling

- Added Pillow to requirements for image processing.
- Updated ZoomApp to use new image paths and improved element interaction.
- Modified MODULE.bazel to reflect the latest Zoom client version.
- Enhanced .gitignore to include temporary files.

d61h6k4 commented on examples/app/zoom_app.py:
```diff
@@ -7,6 +7,7 @@
 
 import logging
 from python.runfiles import runfiles  # pyright: ignore
+from PIL import ImageChops, ImageStat
```
How do you use these functions?

Demontego committed:
Update requirements and clean up ZoomApp imports

- Removed unnecessary comments in requirements_lock.txt for clarity.
- Adjusted requirements.in to maintain consistent package listing.
- Cleaned up imports in zoom_app.py by removing unused ImageChops and ImageStat imports.
